ProductTree/ViewModels/ProductTreeViewModel.cs

```diff
@@ -302,22 +320,25 @@ public async Task Drop(IDropInfo dropInfo)
         }
 
         /// <summary>
-        /// Initialize the <see cref="ICommand"/>
+        /// Initialize the <see cref="ReactiveCommand"/>
         /// </summary>
         protected override void InitializeCommands()
         {
             base.InitializeCommands();
             this.CreateSubscriptionCommand = ReactiveCommand.Create(this.WhenAnyValue(x => x.CanCreateSubscription));
             this.CreateSubscriptionCommand.Subscribe(_ => this.ExecuteCreateSubscriptionCommand());
 
+            this.CopyModelCodeToClipboardCommand = ReactiveCommand.Create(this.WhenAnyValue(x => x.CanCopyToClipboard));
```

Comment:
Same as in the other file, I dont think you need the `CanCopyToClipboard` property.